x/incentive/client/cli/query.go

```diff
@@ -98,3 +100,55 @@ func queryParamsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+func queryRewardPeriodsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "reward-periods",
+		Short: "get active reward periods",
+		Long:  "Get the current set of active incentive reward periods.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			// Query
+			route := fmt.Sprintf("custom/%s/%s", queryRoute, types.QueryGetRewardPeriods)
+			res, _, err := cliCtx.QueryWithData(route, nil)
```

Comment:
```suggestion
			res, height, err := cliCtx.QueryWithData(route, nil)
```

then set the height to the cliCtx with `cliCtx.WithHeight(height)`